build.sbt

```diff
@@ -3,22 +3,16 @@ ThisBuild / homepage := Some(url("https://github.com/avast/scala-server-toolkit"
 ThisBuild / description := "Functional programming toolkit for building server applications in Scala."
 ThisBuild / licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/avast/scala-server-toolkit/master/LICENSE"))
 ThisBuild / developers := List(Developer("jakubjanecek", "Jakub Janecek", "janecek@avast.com", url("https://www.avast.com")))
-ThisBuild / scmInfo := Some(
-  ScmInfo(url("https://github.com/avast/scala-server-toolkit"), "scm:git:git@github.com:avast/scala-server-toolkit.git")
-)
 
 ThisBuild / scalaVersion := "2.13.0"
```

Comment:
btw are we not building this for 2.12? Why?

Reply:
Well, idea that it could only be built against `2.13` maybe :) But I guess it won't fly... we can update that later before first stable release.